Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -104,7 +104,7 @@ public virtual void OnSourceLost(SourceStateEventData eventData)
 
                 if (destroyOnSourceLost)
                 {
-                    if (Application.isEditor)
+                    if (!Application.isPlaying)
```

Comment:
>if (!Application.isPlaying) [](start = 20, length = 27)

No biggie and thanks for spotting it but this seems to fix a different bug. If that is the case I would have rather seeing this change in a separate PR and linking to its own bug.

Reply:
Actually, without this fix the playmode tests will fail, caused by a null reference exception that happens when removing the controller for a simulated hand.